src/main/java/eggventory/logic/parsers/ParseList.java

```diff
@@ -2,31 +2,43 @@
 
 import eggventory.commons.exceptions.InsufficientInfoException;
 import eggventory.logic.commands.Command;
+import eggventory.logic.commands.list.ListPersonCommand;
 import eggventory.logic.commands.list.ListStockCommand;
 import eggventory.logic.commands.list.ListStockTypeCommand;
 import eggventory.commons.enums.CommandType;
 import eggventory.commons.exceptions.BadInputException;
 
 public class ParseList {
 
+    private final String listErrorMessageGeneric = "Usage of list: 'list stock', 'list stocktype all', "
+            + "'list stocktype <Stock Type>', or 'list person'";
+
     private Command processListStock(String input) throws BadInputException {
         String[] inputArr = input.split(" ");
         if (inputArr.length > 1) { // Checking if anything extraneous is after stock
-            throw new BadInputException("Usage of list: 'list stock', 'list stocktype all' or "
-                    + "'list stocktype <Stock Type>'");
+            throw new BadInputException(listErrorMessageGeneric);
```

Comment:
ooh this is great. should refactor other code to do this too

Reply:
We need to create new exception classes...BadInputException is overused...